.github/workflows/approve-flake-lock-prs.yml

```diff
@@ -0,0 +1,16 @@
+# This job applies one approval automatically to the automated `flake.lock` PRs
+# This helps us keep up with the weekly automated PRs, but still requires at least 1
+# human manual approval.
+name: Approve flake.lock PRs (still require 1 human approval)
+permissions:
+  pull-requests: write
+jobs:
+  approve-flake-lock-prs:
+    runs-on: ubuntu-latest
+    if: github.actor == 'github-actions[bot]' && github.event.pull_request.labels.*.name == 'flake.lock automation'
```

Comment:
Would it be possible to also check that only flake files have been modified?

My concern is that the flake.lock automation label can be manually added also for PRs that aren't flake files, so that doesn't provide this guarantee.

Reply:
I think there's probably a way to do that, I'll dig into it!